### PR TITLE
dark mode logo service-mesh-interface fixed

### LINKED ISSUE #1016: [UI] Replace logo with the dark mode version when the website in dark mode.
#### Current State of the Meshery.io site
Screenshot from 2023-02-13 00-53-54

The logo inside that yellow rectangle shown above does not resembles with the dark mode of the website.
link: https://meshery.io/service-mesh-interface

#### Desired State of the Meshery.io site
It should be replaced with the dark mode version of that logo when the website in dark mode.

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by joshboye**
**Description**

This PR fixes #1016 

**Notes for Reviewers**

logo becomes light in dark mode - 

firefox_4EtbYGGsxT

In dark mode - 

firefox_7m2zOnEteh

full page view in dark and light mode-

oR5FQ3dMhv

firefox_lQw3NtOrjA

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_pages/smi.html [hunk 1]
```diff
@@ -76,6 +76,9 @@
                 break;
               case "Kuma":
                 img.setAttribute("src", "assets/images/kuma.svg");
+                img.setAttribute("id", "logo-dark-light");
+                img.setAttribute("data-logo-for-dark", "assets/images/kuma-icon-white.svg");
+                img.setAttribute("data-logo-for-light", "assets/images/kuma.svg");
```
Comment (samyakshah3008) [inline collections/_pages/smi.html:81]: Can we think of better name of image? 
So as to maintain consistency with the image for "light theme"? 
Maybe we can changes both names as `dark-kuma` and `light-kuma`? or any other suggestions? 

Reason being if any developer comes to the codebase, how he will get to know which one is for light theme and which one is for dark? 
For line number 80, naming convention seems good but I think we should change for line number 81 🤔

**Review by samyakshah3008 — APPROVED** [review]
Nice work! ✨  
Added one small comment on "naming for image", please check.

**Comment by samyakshah3008** [comment]
@joshboye can you please attach some screenshots of your updated change? That can help reviewer to review PR better! 😄

File: collections/_pages/smi.html (hunk 1 above)
Comment (joshboye) [reply collections/_pages/smi.html:81]: Sure! 
I will change the names as `kuma-icon-white` for dark theme and `kuma-icon-dark` for light theme. This seems more descriptive?

**Comment by joshboye** [comment]
logo becomes light in dark mode - 
firefox_4EtbYGGsxT

In dark mode - 
firefox_7m2zOnEteh

full view in dark and light mo
oR5FQ3dMhv
de- 
firefox_lQw3NtOrjA

File: collections/_pages/smi.html (hunk 1 above)
Comment (samyakshah3008) [reply collections/_pages/smi.html]: Yes, looks fine now 🚀

**Comment by ritiksaxena124** [comment]
Nice Work @joshboye 🙌, looks fine to me

**Comment by vishalvivekm** [comment]
@joshboye 
Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by joshboye** [comment]
> @joshboye Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

Sure!

**Comment by thisiskaransgit** [comment]
@joshboye 
Let's discuss this in websites call. Add this as agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by leecalcote** [comment]
Good work, @joshboye 👍

**Pull request merged**